persistence-modules/java-cassandra/src/main/java/com/baeldung/cassandra/batch/Application.java

```diff
@@ -0,0 +1,60 @@
+package com.baeldung.cassandra.batch;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.baeldung.cassandra.batch.domain.Product;
+import com.baeldung.cassandra.batch.repository.KeyspaceRepository;
+import com.baeldung.cassandra.batch.repository.ProductRepository;
+import com.datastax.oss.driver.api.core.CqlSession;
+
+public class Application {
```

Comment:
The cassandra example files seem to be unrelated to this article.

Reply:
I was able to remove this. though these were all merged but still showing in my current branch. but after fetching from upstream master had fixed that.